paternoster/paternoster.py

```diff
@@ -168,3 +206,59 @@ def execute(self):
         if status and self._success_msg:
             print(self._success_msg)
         return status
+
+    @staticmethod
+    def prompt(text):
+        """
+        Return user input from a prompt with *text*.
+
+        Works with Python 2 and 3.
+
+        """
+        try:
+            return raw_input(text)
+        except NameError:
+            return input(text)
+
+    @staticmethod
+    def get_input(param):
+        """
+        Return user input for *param*.
+
+        The text for the prompt is taken from `param['prompt']`. You can set
+        additional arguments in `param['prompt_options']`:
+
+        :strip: if `True`: strips user input
+        :empty: if `True`: allows empty input
+        :confirm: if `True` or string: prompt user for confirmation
+        :confirm_error: if string: used on confirmation error
+
+        Raises:
+            ValueError: if input and confirmation do not match.
+
+        """
+        prompt = param.get('prompt')
+        if not isinstance(prompt, six.string_types):
+            prompt = 'Input: '
+        options = param.get('prompt_options', {})
+
+        while True:
+            value = Paternoster.prompt(prompt)
+            if options.get('strip'):
+                value = value.strip()
+            if value or options.get('empty'):
+                break
+
+        if options.get('confirm'):
+            confirmation_prompt = options.get('confirm')
```

Comment:
`str(options.get('confirm', 'Please confirm: '))` looks less cluttered to me, but choose whichever variant you like more.

Reply:
Yes it's cluttered. That is because `prompt` and `confirm` can either take _True_ and use the default prompt, or a (non empty) _string_, to define your own prompt.

Reply:
oh, right.. welp.